### PR TITLE
Additional styling

### PR DESCRIPTION
**Pull request by lydia-zheng**
> Fixed Event Modal button styling
> Fixed Calendar Page responsiveness
> Converted comments in both modals to be table
> Fixed rating field display in Park Modal Description
> Fixed minor bug for date after updating event in Event Modal

### CODE REVIEW

**Comment by gordonzhang17** [comment]
missing styling on the buttons

File: parkhangs-webapp/src/features/modal/ModalEventDetail.css [hunk 1]
```diff
@@ -130,12 +129,58 @@
...
+    border-radius: 3px;
+    color: #000;
+}
+.react-add-to-calendar__button--light{
+
+}
+.dropdown{
+    position: absolute;
+    margin-top: 0.65rem;
+    margin-left: -17px;
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalEventDetail.css:145]: keep to rem

File: parkhangs-webapp/src/features/modal/ModalEventDetail.css [hunk 2]
```diff
@@ -130,12 +129,58 @@
...
+    padding-top: 1rem;
+    padding-right: 0.8rem;
+    padding: 0.5rem;
+    background-color: #f9f9f9;
+    border: 1px solid #aab9d4;
+    border-radius: 3px;
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalEventDetail.css:151]: rem

File: parkhangs-webapp/src/features/modal/ModalEventDetail.css [hunk 3]
```diff
@@ -130,12 +129,58 @@
...
+    display: inline-block;
+    
+    
+}
+.dropdown > ul > li {
+    position: relative;
+    padding: 0.5rem;
+    margin-bottom: 0.5rem;
+    border: 2px solid #aab9d4;
+    border-radius: 3px;
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalEventDetail.css:166]: rem for lines 165, 166

File: parkhangs-webapp/src/features/modal/ModalParkDetail.js [hunk 4]
```diff
@@ -134,48 +134,48 @@ class ModalParkDetail extends React.Component {
...
-                                Park advisories?: {park.hasAdvisories ? "yes" : "no"}
+                                <b>Park advisories?:</b> {park.hasAdvisories ? "yes" : "no"}
                             </div>
                             <div>
-                                Special Features?: {park.hasSpecialFeatures ? "yes" : "no"}
+                                <b>Special Features?:</b> {park.hasSpecialFeatures ? "yes" : "no"}
                             </div>
                             <div>
-                                Rating: {park.rating}
+                                <b>Rating:</b> {!!averageRating ? averageRating: "N/A"}
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalParkDetail.js:175]: Can we say something like "This park hasn't been rated yet"? isn't of N/A

File: parkhangs-webapp/src/features/modal/ModalParkDetail.js [hunk 5]
```diff
@@ -134,48 +134,48 @@ class ModalParkDetail extends React.Component {
...
                             </div>
                             <div>
-                                Number of Favorites: {park.favoritesCount || 0}
+                                <b>Number of Favorites:</b> {park.favoritesCount || 0}
```
Comment (gordonzhang17) [inline parkhangs-webapp/src/features/modal/ModalParkDetail.js:178]: "Number of users that like this park"

File: parkhangs-webapp/src/features/modal/ModalEventDetail.css (hunk 3 above)
Comment (lydia-zheng) [reply parkhangs-webapp/src/features/modal/ModalEventDetail.css:166]: Borders can be static value so I don’t need to switch to rem, or that’s my understanding.

File: parkhangs-webapp/src/features/modal/ModalParkDetail.js (hunk 5 above)
Comment (lydia-zheng) [reply parkhangs-webapp/src/features/modal/ModalParkDetail.js:178]: I think the current is clear enough for intent

**Pull request merged**